libfuzzer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stable-slim
+RUN apt-get update && \
+    apt-get install -fy clang curl wget libfuzzer-7-dev libc6-dbg && \
+    # Install Rust
```

Comment:
I would recommend we split this into three different fuzzers: `libfuzzer`, `cargo-fuzz` and `go-fuzz`. what do you think?

Reply:
sounds good!

Reply:
Done!